intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorVersions.kt

```diff
@@ -0,0 +1,9 @@
+package com.jetbrains.plugin.structure.ktor
+
+val KTOR_VERSIONS = listOf(
```

Comment:
how you will update that set?

and maybe setOf()?

Reply:
`setOf()` -- Agree, will fix.

Regarding updates -- I think this should be updated each time a new version of Ktor releases (approx. each month). Does this seem possible?

Reply:
too hard. I guess You will forget to update that field :(

Reply:
Do we have any other options?

Reply:
validate ktor version on ktor-feature-generator?